packages/block-editor/src/hooks/index.js

```diff
@@ -88,3 +88,4 @@ export { getTypographyClassesAndStyles } from './use-typography-props';
 export { getGapCSSValue } from './gap';
 export { useCachedTruthy } from './use-cached-truthy';
 export { useZoomOut } from './use-zoom-out';
+export { useUpdateBlockVariationOverridesWithConfig } from './block-style-variation';
```

Comment:
```suggestion
export { useUpdateBlockStyleVariationOverridesWithConfig } from './block-style-variation';
```

If adopting the earlier suggestion to keep the delineation between block style variations and block variations clear, we'll need this and the other references below updated too.